shared/common-adapters/usernames/container.js

```diff
@@ -43,11 +43,10 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
     onUsernameClicked = ownProps.onUsernameClicked
   }
 
-  return {
-    ...ownProps,
-    onUsernameClicked,
-    users: userData,
-  }
+  // $FlowIssue for some reason, Flow seems to think that onUsernameClicked can be "tracker" or "profile", even though we've explicitly overridden those
```

Comment:
I think this is becuase of the `else` block above. maybe you can do a type check on `ownProps.onUsernameClicked` to convince flow that it's a function

Reply:
Interestingly, the following results in the same error:
```js
  let onUsernameClicked
  if (ownProps.onUsernameClicked === 'tracker') {
    onUsernameClicked = dispatchProps.onOpenTracker
  } else if (ownProps.onUsernameClicked === 'profile') {
    onUsernameClicked = dispatchProps.onOpenProfile
  } else if (typeof onUsernameClicked === 'function') {
    onUsernameClicked = ownProps.onUsernameClicked
  }

  // Error here
  const props: Props = {...ownProps, onUsernameClicked, users: userData}
```

Gonna leave the `$FlowIssue` because I'm not sure what else to do?